doc/ref_model/chapters/chapter04.md

```diff
@@ -263,63 +263,178 @@ Table 4-10 shows performance measurement capabilities.
+
+<a name="4.2.1.2"></a>
+#### 4.2.1.2 High Performance Profile
+
+The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
 
 <a name="4.2.2"></a>
-### 4.2.2 Virtual Network Interface Specifications
+### 4.2.2 Profiles Specifications & Capability Mapping
+
+| Ref | Capability  | Basic | High Performance | Notes |
+|-----|---------|----------|----------|--------|
+| e.cap.006 | CPU pinning | No | Yes | Exposed performance capabilities as per Table 4-2. |
+| e.cap.007 | NUMA alignment  | No | Yes | |
+| e.cap.013 | SR-IOV over PCI-PT  | No   | Yes | |
+| i.cap.018 | Huge page support  | No  | Yes | Internal performance capabilities as per Table 4-7. |
+| e.cap.018 | Simultaneous Multithreading (SMT) | Yes | Yes | |
+| e.cap.019 | vSwitch Optimisation (DPDK) | No | Yes| |
+| e.cap.020 | CPU Architecture | \<value> | \<value> | Values such as x64, ARM, etc. |
+| e.cap.021 | Host Operating System (OS) | \<value> | \<value> | Values such as a specific Linux version, Windows version, etc. |
+| e.cap.022 | Virtualisation Infrastructure Layer<sup>1</sup> | \<value> | \<value> | Values such as KVM, Hyper-V, Kubernetes, etc. when relevant, depending on technology. |
+| i.cap.019 | CPU Clock Speed | \<value> | \<value> | Specifies the Cloud Infrastructure CPU Clock Speed (in GHz). |
+| i.cap.020 | Storage encryption | Yes | Yes | Specifies whether the Cloud Infrastructure supports storage encryption. |
+> <sup>1</sup> See [Figure 5-1](./chapter05.md#5.1).
+
+<a name="4.2.3"></a>
+### 4.2.3 Profile Extensions
+
+Profile Extensions represent small deviations from or further qualification of the profiles that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Profile Extensions provide workloads a more granular control over what infrastructure they can run on.
+
+| Profile Extension Name | Mnemonic | Applicable to Basic Profile | Applicable to High Performance Profile | Description | Notes |
+|----|----|----|----|----|----|
+| Compute Intensive High-performance CPU | compute-high-perf-cpu | ❌ | ✅ | Nodes that have predictable computing performance and higher clock speeds. | May use vanilla VIM/K8S scheduling instead. |
+| Storage Intensive High-performance storage | storage-high-perf | ❌ | ✅ | Nodes that have low storage latency and/or high storage IOPS |  |
+| Compute Intensive High memory | compute-high-memory | ❌ | ✅ | Nodes that have high amounts of RAM. | May use vanilla VIM/K8S scheduling instead. |
+| Compute Intensive GPU | compute-gpu | ❌ | ✅ | for compute intensive Workloads that requires GPU compute resource on the node | May use Node Feature Discovery. |
```

Comment:
@pgoyal01 Typos in notes description -> Capital letter missing in last ligns

Reply:
@karinesevilla Done - Thanks.